spec/orders_spec.rb

```diff
@@ -0,0 +1,151 @@
+# encoding: utf-8
+
+require 'spec_helper'
+require 'active_support/core_ext/string/strip'
+
+describe FakeBraspag::Orders do
+  before do
+    Order.connection.flushdb
+  end
+
+  let(:order_params) do
+    {
+      'merchantId' => '{E8D92C40-BDA5-C19F-5C4B-F3504A0CFE80}',
+      'order' => '',
+      'orderId' => '994567',
+      'customerName' => 'Antonio Carlos',
+      'amount' => '18,36',
+      'paymentMethod' => '42',
+      'holder' => 'Antonio Carlos',
+      'cardNumber' => '5555555555554444',
+      'expiration' => '02/23',
+      'securityCode' => '123',
+      'numberPayments' => '1',
+      'typePayment' => '0'
+    }
```

Comment:
replace this to [HashWithIndifferentAcces](http://api.rubyonrails.org/classes/ActiveSupport/HashWithIndifferentAccess.html) is an option?

otherwise <3 <3 <3